policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,76 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(period).gov.states.ar.tax.income.deductions.itemized
+        p_ded = parameters(period).gov.irs.deductions
+
+        agi = tax_unit("adjusted_gross_income", period)
+        head = person("is_tax_unit_head", period)
+        person_agi = person("adjusted_gross_income_person", period)
+        total_person_agi = tax_unit.sum(person_agi * head)
+
+        # Less salt deduction
+        deductions = [
+            deduction
+            for deduction in p_ded.itemized_deductions
+            if deduction
+            not in [
+                "salt_deduction",
+                "medical_expense_deduction",
+            ]
+        ]
+        less_salt_deds = add(tax_unit, period, deductions)
+
+        # Real estate tax + Personal property tax
+        real_estate_deds = add(tax_unit, period, ["real_estate_taxes"])
+
+        # Post-secondary Education Tuition Deduction
+        tuition_deds = tax_unit(
+            "ar_post_secondary_education_tuition_deductions", period
+        )
+
+        # Limitation on several items
+        # Medical and Dental Expense
+        medical_expenses = add(tax_unit, period, ["medical_expense"])
+        medical_deds = max_(
+            0,
+            medical_expenses - p.expense_rate.medical * agi,
```

Comment:
double check if we multiply by federal or state AGI

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/89811054/efc17310-3a29-4269-bcd9-1fe852db1ce0)

The agi used in the medical deduction and the miscellaneous deductions are both from line 25 A and B in the form AR1000F as above. I assume that it should be state agi since it considers the state adjustment, am I thinking correctly about this? If it's the state agi, does it mean that I should use tax_unit("ar_taxable_income", period) instead? Thank you!

Reply:
we need to create an `ar_agi.py`

Reply:
no formula yet, we will add it later